src/client/app/containers/import.ts

```diff
@@ -2,10 +2,14 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
 import * as fromRoot from '../reducers';
 import { Store } from '@ngrx/store';
 import { UI, Snippet, GitHub } from '../actions';
-import { AI, storage } from '../helpers';
+import { UIEffects } from '../effects/ui';
+import { environment, AI, storage } from '../helpers';
+import { Request, ResponseTypes } from '../services';
 import { Strings } from '../strings';
 import { isEmpty } from 'lodash';
 
+const viewUrlSettings = 'VIEW_URL_SETTINGS';
```

Comment:
This will also fit better, naming-wise, with doing a base-64-string import, for example